pkg/infoschema/infoschema_v2.go

```diff
@@ -646,6 +646,25 @@ func (is *infoschemaV2) TableByID(ctx context.Context, id int64) (val table.Tabl
 	return ret, true
 }
 
+func (is *infoschemaV2) SchemaNameByTableID(tableID int64) (schemaName pmodel.CIStr, ok bool) {
+	if !tableIDIsValid(tableID) {
```

Comment:
also extract a `tableItemByID` from `TableByID` and reuse it.

Reply:
Updated.